src/Facebook/InstantArticles/Validators/Type.php

```diff
@@ -367,19 +367,48 @@ public static function enforceWithin($value, $universe)
 
     private static function throwNotWithinException($value, $universe)
     {
-      // stringify the $value parameter
-        ob_start();
-        var_dump($value);
-        $value_str = ob_get_clean();
-
-      // stringify the $universe parameter
-        ob_start();
-        var_dump($universe);
-        $universe_str = ob_get_clean();
+        $value_str = self::stringify($value);
+        $universe_str = self::stringify($universe);
 
         throw new \InvalidArgumentException(
             "Method expects this value \n----[\n".$value_str."]----\n".
             " to be within this universe of values \n====[\n".$universe_str."]===="
         );
     }
+
+    /**
+     * Checks the thext if it is empty.
+     * "" => true
+     * "    " => true
+     * "\n" => true
+     * "a" => false
+     * "  a  " => false
+     *
+     * @param string $text The text that will be checked.
+     * @return true if empty, false otherwise.
+     */
+    public static function isTextEmpty($text)
+    {
+        if (!isset($text) || $text === null || !self::is($text, self::STRING)) {
+            return true;
+        }
+        // Stripes empty spaces, &nbsp;, <br/>, new lines
+        $text = strip_tags($text);
+        $text = preg_replace("/[\r\n\s]+/", "", $text);
+        $text = str_replace("&nbsp;", "", $text);
+
+        return empty($text);
+    }
+
+    /**
+     * Auxiliary method that stringify an object as var_dump does.
+     * @return string $object var_dump result.
+     */
+    public static function stringify($object)
+    {
+          // stringify the $object parameter
+          ob_start();
```

Comment:
Output buffers are slow, and that looks pretty hacky too. Can we evaluate using var_export instead (which allows for returning a string instead of outputting)?